source/nodejs/adaptivecards/src/scss/adaptivecards-base.scss

```diff
@@ -13,6 +13,7 @@ $font-family: 'Segoe UI', sans-serif;
 
             &:hover {
                 background-color: $selectable-hover-background !important;
+				cursor: pointer;
```

Comment:
We then probably don't need `{cursor: pointer}` [here](https://github.com/microsoft/AdaptiveCards/blob/951131c964682b6c0d32ebb25528b10ff5ec186f/source/nodejs/adaptivecards/src/scss/adaptivecards-base.scss#L25)
![image](https://user-images.githubusercontent.com/21356912/160933499-85f1606f-fff6-4384-87de-7063c6b7e4c3.png)

since we'll have it on `.ac-selectable`

Reply:
@beervoley Don't we need it here because we only have it on `ac-container.ac-selectable` and `ac-columnSet.ac-selectable` above?

Reply:
oh, you're right - my mistake. We still need it :)
I didn't read the code correctly :)

Reply:
@paulcam206 may be we should extract one liner `.ac-selectable {cursor: pointer}`? since it's natural for selectable items to have this?:)

Reply:
@paulcam206 Did you have any thoughts on this?